tests/scripts/run-xunit-perf.cmd

```diff
@@ -252,6 +277,12 @@ rem ****************************************************************************
     shift
     goto :parse_command_line_arguments
   )
+  IF /I [%~1] == [-sliceDir] (
+    set SLICE_DIR=%~2
```

Comment:
Each slice can run multiple directories worth of tests according to the config file.  What does the -sliceDir parameter present?  I thought I saw that the parameter passed to each job was a slice number.

Reply:
sliceDir tells run-xunit-perf.cmd where run-xunit-slice.py wrote out intermediate files.